BaseClasses.py

```diff
@@ -717,9 +717,10 @@ def can_reach(self,
                 spot = self.multiworld.get_region(spot, player)
         return spot.can_reach(self)
 
-    def sweep_for_events(self, key_only: bool = False, locations: Optional[Iterable[Location]] = None) -> None:
+    def sweep_for_events(self, key_only: bool = False, locations: Optional[Iterable[Location]] = None,
```

Comment:
I prefer my core changes in seperate PRs.

Reply:
Removed